File: src/main/java/gg/mineral/practice/util/messages/ErrorMessage.java
package gg.mineral.practice.util.messages;

import org.bukkit.entity.Player;

public class ErrorMessage extends Message {
	public String addition;

	public ErrorMessage(String s) {
		message = s;
		formatMessage();
	}

	private void formatMessage() {
		this.addition = CC.D_RED + "✖ Error ✖ " + CC.RED;
		message = addition + message;
	}

	@Override
	public void send(Player p) {
		p.sendMessage(message);
	}
}

File: src/main/kotlin/gg/mineral/practice/util/messages/ErrorMessage.kt
package gg.mineral.practice.util.messages

class ErrorMessage(s: String) : Message(s) {
    override fun clone() = ErrorMessage(messageBuilder.toString())

    override fun prepend(builder: StringBuilder): StringBuilder = builder.insert(0, CC.D_RED + "✖ Error ✖ " + CC.RED)
}
